### ISSUE TITLE
zsh history stdout is aggressively cleaned by tilda

### ISSUE THREAD

**Issue by rjshrjndrn** [lanoxx/tilda#378]
Hi,
Thank you for the awesome terminal It's my daily driver.
But I've got a small concern.
Tilda is not keeping any history, say I have a 1000 line stdout, and when I scrollback may be 20 lines will be there. But all others will be gone. This is a major problem if you just logged out of an ssh session or things like that.

Thank you.

**Comment by rjshrjndrn** [comment]
```

**Comment by rjshrjndrn** [comment]
I think I found the solution. Scroll on output was disabled. Thank you.

**Comment by rjshrjndrn** [comment]
Sorry that was not the solution.

steps to recreate the issue
1.` while true; do ls;done`
press ctrl+c after sometime
2. clear <enter>
do clear twice

and scroll up.

**Comment by MatthewScholefield** [comment]
Yes, as can be seen [here](https://askubuntu.com/questions/434240/ctrll-in-terminal/434243#comment1882071_434243), this is roughly the expected behavior. Note that if anything the only bug is that tilda doesn't clear the last 10 lines or so when getting cleared via `clear`.
